robosat/metrics.py

```diff
@@ -27,7 +27,14 @@ def add(self, actual, predicted):
           predicted: the predicted labels.
         """
 
-        matrix = sklearn.metrics.confusion_matrix(actual, predicted, labels=self.labels)
+        confusion = predicted / actual
+
+        tn = torch.sum(torch.isnan(confusion)).item()
+        fn = torch.sum(confusion == float("inf")).item()
+        fp = torch.sum(confusion == 0).item()
+        tp = torch.sum(confusion == 1).item()
+
+        matrix = np.array([[tn, fn], [fp, tp]])
```

Comment:
Did you run this and compared it against the sklearn confusion matrix side by side?